service/test/variables_test.go

```diff
@@ -840,6 +840,9 @@ func TestEvalExpression(t *testing.T) {
 		{"ni8 << 8", false, "0", "0", "int8", nil},
 		{"ni8 >> 1", false, "-3", "-3", "int8", nil},
 		{"bytearray[0] * bytearray[0]", false, "144", "144", "uint8", nil},
+
+		// pretty printing special types
+		{"tim1", false, `time.Time(1977-05-25T18:00:00Z)…`, `time.Time(1977-05-25T18:00:00Z)…`, "time.Time", nil},
```

Comment:
Is this always GMT+0000 timezone?
I can already imagine users wanting to see their timezone or be able to configure the timezone.

Reply:
I'm not sure we can do any better. We could look up the time.Time.Location.name field, and try to load it with time.LoadLocation, maybe that's good enough but if the program created the timezone using time.FixedZone the name could be anything.

Reply:
Possibly in the future we could let users specify this?

Reply:
How would they specify it?

Reply:
I was thinking more along the lines of a custom format print command or something. Sorry, kinda thinking out loud without concrete ideas.

Reply:
I've changed it to load the location, if there is one and print it. If we want to go with the scripting thing then picking the location becomes part of that.